src/BloomBrowserUI/lib/long-press/jquery.longpress.js

```diff
@@ -233,8 +233,12 @@
         if (isTextArea()) {
             setTextAreaCaretPosition(activeElement, textAreaCaretPosition);
         } else {
-            EditableDivUtils.makeSelectionIn(activeElement, storedOffset, null, true);
-            setFocusDelayed();
```

Comment:
The delayed focus was part of the original code taken from the network.  I have no idea why they thought it was needed.  Testing the code on my machine seemed to make no difference whether it was delayed (with the selection setting also delayed) or done immediately.